glean-core/android/src/main/java/mozilla/telemetry/glean/debug/GleanDebugActivity.kt

```diff
@@ -20,9 +20,9 @@ import mozilla.components.support.base.log.logger.Logger
  * https://developer.android.com/studio/command-line/adb#am
  */
 class GleanDebugActivity : Activity() {
-    private val logger = Logger("glean/GleanDebugActivity")
-
     companion object {
+        private const val LOG_TAG = "GleanDebugActivity"
```

Comment:
Ok, this wouldn't fit in 23 characters. Maybe we can change this to "glean/DebugActivity"?